specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-09-01/DataConnectors.json

```diff
@@ -0,0 +1,1696 @@
+          "description": "The DCR related properties.",
+          "$ref": "#/definitions/DCRConfiguration"
+        },
+        "isActive": {
+          "description": "Indicates whether the connector is active or not.",
+          "type": "boolean"
+        },
+        "dataType": {
+          "description": "The Log Analytics table destination.",
+          "type": "string"
+        },
+        "response": {
+          "description": "The response configuration.",
+          "$ref": "#/definitions/CcpResponseConfig"
+        },
+        "paging": {
+          "description": "The paging configuration.",
+          "$ref": "#/definitions/RestApiPollerRequestPagingConfig"
+        },
+        "addOnAttributes": {
+          "description": "The add on attributes. The key name will become attribute name (a column) and the value will become the attribute value in the payload.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      },
+      "type": "object",
+      "required": [
+        "connectorDefinitionName",
+        "auth",
+        "request"
+      ]
+    },
+    "RestApiPollerRequestConfig": {
+      "type": "object",
+      "description": "The request configuration.",
+      "properties": {
+        "apiEndpoint": {
+          "description": "The API endpoint.",
+          "type": "string"
+        },
+        "rateLimitQPS": {
+          "description": "The Rate limit queries per second for the request..",
+          "type": "integer",
+          "x-nullable": true,
+          "format": "int32"
+        },
+        "queryWindowInMin": {
+          "description": "The query window in minutes for the request.",
+          "type": "integer",
+          "x-nullable": true,
+          "format": "int32"
+        },
+        "httpMethod": {
+          "description": "The HTTP method, default value GET.",
+          "enum": [
+            "GET",
+            "POST",
+            "PUT",
+            "DELETE"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "httpMethodVerb"
+          }
+        },
+        "queryTimeFormat": {
+          "description": "The query time format. A remote server can have a query to pull data from range 'start' to 'end'. This property indicate what is the expected time format the remote server know to parse.",
+          "example": "UnixTimestamp, UnixTimestampInMills, or more specific formats, for example: yyyy-MM-ddTHH:mm:ssZ",
+          "type": "string"
+        },
+        "retryCount": {
+          "description": "The retry count.",
+          "type": "integer",
+          "x-nullable": true,
+          "format": "int32"
+        },
+        "timeoutInSeconds": {
+          "description": "The timeout in seconds.",
+          "type": "integer",
+          "x-nullable": true,
+          "format": "int32"
+        },
+        "isPostPayloadJson": {
```

Comment:
>isPostPayloadJson [](http://example.com/codeflow?start=9&length=17)

In place of having this boolean you can replace it with an enum like payloadFormat with values [Json, Form]